terraform/roles.tf

```diff
@@ -288,3 +288,58 @@ resource aws_iam_role_policy_attachment ec2_readonly {
   role       = aws_iam_role.caascad_operator.name
   policy_arn = "arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess"
 }
+
+data aws_iam_policy_document caascad_monitoring_cloud_services_assumerole_policy {
```

Comment:
Should i reuse the policy document `caascad_operator_assumerole_policy` ?
I don't know because we might want to update one without updating the other.

Reply:
Assume role policy will not change -> reuse the same.